mypy/build.py

```diff
@@ -825,8 +825,20 @@ def _load_json_file(file: str, manager: BuildManager,
         manager.log(log_error + file)
         return None
     manager.trace(log_sucess + data.rstrip())
-    result = json.loads(data)  # TODO: Errors
-    return result
+    try:
+        result = json.loads(data)
+    except ValueError:  # TODO: JSONDecodeError in 3.5
+        manager.errors.report(0, 0, "Error reading JSON file {file};"
```

Comment:
Hm, I tried this, and it reports
```
setup.cfg:0: error: Error reading JSON file _.meta.json; you likely have a bad cache.
 Try removing the .mypy_cache directory and run mypy again.
```
Why does it report `setup.cfg` as the filename? And why the space in front of `Try`?